### PR TITLE
replaced learning home page placeholder images

### PR DESCRIPTION
**Pull request by ZhuZhuThanou**
### Description
Added new images as prescribed in Figma for the learning contents. Images for real life stories sections are not included as the final design is  not done. 

List of proposed changes:
- changed the MediaCard component to fit the images but makes the placeholder image does not look great
- new image for from-work-to-retirement MediaCard
- new image for planning-to-save-for-retirement MediaCard
- new image for retirement-income-sources MediaCard
- new image for rule-of-thumb-public-pensions MediaCard
- new image for when-to-take-your-pensions MediaCard

### What to test for/How to test
- npm i
- npm run dev
- go to [http://localhost:3000/en/learn](http://localhost:3000/en/learn)

### Additional Notes

### CODE REVIEW

File: frontend/public/assets/main-sources-of-income.svg
[the remark this answers is not in the document]
Comment (sebastien-comeau) [reply frontend/public/assets/main-sources-of-income.svg:1]: https://help.streamlinehq.com/en/articles/5752796-what-is-the-difference-between-svg-and-png

[the remark this answers is not in the document]
Comment (gregory-j-baker) [reply frontend/public/assets/main-sources-of-income.svg:1]: image

Yah.. a bunch of these being downloaded by the client is definitely gonna be a problem. 🤣

[the remark this answers is not in the document]
Comment (sebastien-comeau) [reply frontend/public/assets/main-sources-of-income.svg:1]: Resize them to the container max size of `1536px` and keep aspect-ratio.

[the remark this answers is not in the document]
Comment (sebastien-comeau) [reply frontend/public/assets/main-sources-of-income.svg:1]: Those images are JPEG.
image

[the remark this answers is not in the document]
Comment (ZhuZhuThanou) [reply frontend/public/assets/main-sources-of-income.svg:1]: yeah. I wasn't sure. For photos, what's the standard in ESDC? JPEG?

[the remark this answers is not in the document]
Comment (sebastien-comeau) [reply frontend/public/assets/main-sources-of-income.svg:1]: Those picture are JPEGs.

[the remark this answers is not in the document]
Comment (ZhuZhuThanou) [reply frontend/public/assets/main-sources-of-income.svg:1]: changed the photos to 1536px width and keep aspect-ratio

**Review by sebastien-comeau — APPROVED** [review]

**Review by Fbasham — APPROVED** [review]

**Pull request merged**